tools/pythonpkg/setup.py

```diff
@@ -206,7 +206,9 @@ def open_utf8(fpath, flags):
 for ext in extensions:
     define_macros.append(('DUCKDB_EXTENSION_{}_LINKED'.format(ext.upper()), None))
 
-define_macros.extend([('DUCKDB_EXTENSION_AUTOLOAD_DEFAULT', '1'), ('DUCKDB_EXTENSION_AUTOINSTALL_DEFAULT', '1')])
+if not is_pyodide:
+    # currently pyodide environment is not compatible with dynamic extension loading
+    define_macros.extend([('DUCKDB_EXTENSION_AUTOLOAD_DEFAULT', '1'), ('DUCKDB_EXTENSION_AUTOINSTALL_DEFAULT', '1')])
```

Comment:
Hm, shouldn't autoload still be enabled? I would like the parquet extension to be loaded (for example) when I query a parquet file using code like `SELECT * FROM 'file.parquet'`.

Reply:
Extension logic in duckdb is basically:
* statically linked extensions, those are Loaded at startup
* dynamically linked extensions, when usage is detected at runtime, are possibly autoinstalled and autoloaded depending on the flags.

With those flag off, parquet will still be statically linked and loaded as part of the startup, but it avoids other extensions (say azure) to be pulled in when they are not available.

Reply:
Got it, thanks!